Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    type: mobile app
+    name: The Podcast App
+    version: "6.49.0"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'TREBLE/1.0'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Treble
+    version: "1.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'TREBLE/2.1'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Treble
+    version: "2.1"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Turtlecast/0.7.7; (Android 13); +https://www.turtlecast.app'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: Turtlecast
+    version: "0.7.7"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'TURTLECAST/0.5.5'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Turtlecast
+    version: "0.5.5"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Ubook Player'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Ubook Player
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'VictorReader_4-4-2'
+  os: [ ]
+  client:
+    type: mobile app
+    name: VictorReader
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'VictorReader Stream 503 ca-en Heather Ryan 4.8.15'
+  os: [ ]
+  client:
+    type: mobile app
+    name: VictorReader
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'VictorReader Stream Trek ca-en Sharon Ryan 2.1.31'
+  os: [ ]
+  client:
+    type: mobile app
+    name: VictorReader
+    version: ""
+  device:
+    type: smartphone
+    brand: Olympia
+    model: 'Trek'
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Vodacast/1.2.2 (Linux;Android 12) ExoPlayerLib/2.10.4'
+  os:
+    name: Android
+    version: "12"
+    platform: ""
+  client:
+    type: mobile app
+    name: Vodacast
+    version: "1.2.2"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Vodacast/6 CFNetwork/1404.0.5 Darwin/22.3.0'
+  os:
+    name: iOS
+    version: "16.3"
+    platform: ""
+  client:
+    type: mobile app
+    name: Vodacast
+    version: "6"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'WynkMusic/3.42.0.21 (Linux;Android 13) ExoPlayerLib/2.11.7'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: WynkMusic
```

Comment:
Based on https://wynk.in/music and https://play.google.com/store/apps/details?id=com.bsbportal.music&hl=en_US it should be `Wynk Music` with a space